### DISCUSSION TITLE
靈能的挑戰: 大家對於玄學的看法

### DISCUSSION THREAD

**Discussion by 112121py** [1101-GS4719-C/Python-Community-GS4719-C#2159] [General]
這是一個youtuber跟東森超視合作出的一個網路節目，是一場靈能選秀，就每一集都會讓參賽者透過一些物品或是人物來「算」出他的經歷或是找到指定的目標。
蠻多人對這個節目的看法就是，阿就是看一群小丑在自打嘴巴之類的。我個人是覺得，就算命、占卜、奇門遁甲這類的東西，我是不會到不完全相信，但也不會全盤接收xd。我覺得有時候就像大考前會去拜拜求個心安的那種感覺，你知道以科學的角度那可能沒什麼用，不過還是會去，另一個例子就是綠色乖乖，(但這個就真的很玄了不得不信阿www)
總之，個人秉持一個不要太著迷，但用來警惕自己也是一個還不錯的事情，歡迎大家分享對於這些的看法~
畢竟~~科學的盡頭有座山，叫老山~~
以下提供幾個相關影片，其實還有蠻多，可以去吃瓜看看w
![靈能的挑戰EP1](https://youtu.be/J3uTxdbA-Gk?si=CnBqFYYmTZQRBohG)
![另一方對節目的看法](https://youtu.be/cO6V6cdIOhw?si=edFcRLMMJlTyjaDJ)
![最大贏家](https://youtu.be/B4eKbnDo_Bc?si=JwU8Zei8NCx5jFIQ)

**Comment by zhilin93517** [comment]
我很害怕玄學，對我而言，有時候數學學一學會變成玄學，正面都是符號，但全部都看不懂，有的符號甚至連念都不會唸。因此我認為我對玄學是恐懼的。
